examples/additive_manufacturing/sintering_physics/render_rollout.py

```diff
@@ -0,0 +1,495 @@
+# ignore_header_test
+# ruff: noqa: E402
+
+# © Copyright 2023 HP Development Company, L.P.
+# SPDX-FileCopyrightText: Copyright (c) 2023 - 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ============================================================================
+"""Simple matplotlib rendering of a rollout prediction against ground truth.
+
+Usage (from parent directory):
+
+`python -m learning_to_simulate.render_rollout --rollout_path={OUTPUT_PATH}/rollout_test_1.json`
+
+Where {OUTPUT_PATH} is the output path passed to `train.py` in "eval_rollout"
+mode.
+
+It may require installing Tkinter with `sudo apt-get install python3.7-tk`.
+
+"""
+
+import os, json
+import numpy as np
+from absl import app
+
+import matplotlib.pyplot as plt
+import mpl_toolkits.mplot3d.axes3d as p3
+from matplotlib import animation
+
+from modulus.distributed.manager import DistributedManager
+from modulus.launch.logging import (
+    LaunchLogger,
+    PythonLogger,
+    initialize_wandb,
+    initialize_mlflow,
+    RankZeroLoggingWrapper,
+)
+
+from constants import Constants
+
+
+C = Constants()
```

Comment:
Can we switch to Hydra configs?

Reply:
Let's address this in another PR: https://github.com/NVIDIA/modulus/issues/404

Reply:
Solved